.circleci/config.yml

```diff
@@ -13,34 +13,43 @@ jobs:
       - checkout
       - run: sudo apt install -y golint
       # specify any bash command here prefixed with `run: `
+      - run: go run ./pkg/intel/metadata/manifest/internal/manifestcodegen/cmd/manifestcodegen -check ./pkg/intel/metadata/manifest ./pkg/intel/metadata/manifest/bootpolicy ./pkg/intel/metadata/manifest/key
```

Comment:
@xaionaro This line fails with:

> 2021/02/17 07:50:38 an error: unable to analyze path './pkg/intel/metadata/manifest': unable to get the package from directory './pkg/intel/metadata/manifest': unable to open go directory: unable to import package 'github.com/9elements/converged-security-suite/pkg/intel/metadata/manifest' (in: '/go/src/github.com/9elements/converged-security-suite/pkg/intel/metadata/manifest'): type-checking package "github.com/9elements/converged-security-suite/pkg/intel/metadata/manifest" failed (/go/src/github.com/9elements/converged-security-suite/pkg/intel/metadata/manifest/hash.go:5:8: could not import github.com/google/go-tpm/tpm2 (cannot find package "github.com/google/go-tpm/tpm2" in any of:
	/usr/local/go/src/github.com/google/go-tpm/tpm2 (from $GOROOT)
	/go/src/github.com/google/go-tpm/tpm2 (from $GOPATH)))
2021/02/17 07:50:38 an error: unable to analyze path './pkg/intel/metadata/manifest/bootpolicy': unable to get the package from directory './pkg/intel/metadata/manifest/bootpolicy': unable to open go directory: unable to import package 'github.com/9elements/converged-security-suite/pkg/intel/metadata/manifest/bootpolicy' (in: '/go/src/github.com/9elements/converged-security-suite/pkg/intel/metadata/manifest/bootpolicy'): type-checking package "github.com/9elements/converged-security-suite/pkg/intel/metadata/manifest/bootpolicy" failed (/go/src/github.com/9elements/converged-security-suite/pkg/intel/metadata/manifest/bootpolicy/bpmh.go:6:2: could not import github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/manifest (cannot find package "github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/manifest" in any of:
	/usr/local/go/src/github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/manifest (from $GOROOT)
	/go/src/github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/manifest (from $GOPATH)))
2021/02/17 07:50:38 an error: unable to analyze path './pkg/intel/metadata/manifest/key': unable to get the package from directory './pkg/intel/metadata/manifest/key': unable to open go directory: unable to import package 'github.com/9elements/converged-security-suite/pkg/intel/metadata/manifest/key' (in: '/go/src/github.com/9elements/converged-security-suite/pkg/intel/metadata/manifest/key'): type-checking package "github.com/9elements/converged-security-suite/pkg/intel/metadata/manifest/key" failed (/go/src/github.com/9elements/converged-security-suite/pkg/intel/metadata/manifest/key/hash.go:9:2: could not import github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/manifest (cannot find package "github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/manifest" in any of:
	/usr/local/go/src/github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/manifest (from $GOROOT)
	/go/src/github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/manifest (from $GOPATH)))
exit status 1

Reply:
Can we fix this first? I'll be able to take a look after that.
```
xaionaro@void:~/go/src/github.com/9elements/converged-security-suite-private/pkg/intel/metadata/manifest$ go run ./internal/manifestcodegen/cmd/manifestcodegen/ -check . ./bootpolicy ./key
2021/02/17 09:34:35 an error: file '/home/xaionaro/go/src/github.com/9elements/converged-security-suite-private/pkg/intel/metadata/manifest/hash_manifestcodegen.go' is not up-to-date; please run command: go run github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/manifest/internal/manifestcodegen/cmd/manifestcodegen github.com/9elements/converged-security-suite-private/pkg/intel/metadata/manifest
2021/02/17 09:34:35 an error: file '/home/xaionaro/go/src/github.com/9elements/converged-security-suite-private/pkg/intel/metadata/manifest/bootpolicy/bpmh_manifestcodegen.go' is not up-to-date; please run command: go run github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/manifest/internal/manifestcodegen/cmd/manifestcodegen github.com/9elements/converged-security-suite-private/pkg/intel/metadata/manifest/bootpolicy
2021/02/17 09:34:36 an error: unable to analyze path './key': unable to get the package from directory './key': unable to open go directory: unable to get package info: /home/xaionaro/go/src/github.com/9elements/converged-security-suite-private/pkg/intel/metadata/manifest/key/manifest.go:53:40: cannot use algo (variable of type tpm2.Algorithm) as manifest.TPMAlgorithm value in argument to m.KeyAndSignature.SetSignature
exit status 1
```

Reply:
@ChriMarMe : OK, I'll take a look today-tomorrow. I forgot I won't forget :(

Reply:
No worries :)

Reply:
@ChriMarMe : Fixed.

Reply:
Stop. It still fails in CircleCI. Investigating...

Reply:
OK, now it is fixed...